src/__tests__/to-have-form-values.js

```diff
@@ -331,3 +349,5 @@ function renderSelectMultiple(name, label, options, value = []) {
 function renderList(items, mapper) {
   return items.map(mapper).join('')
 }
+
+/* eslint max-lines-per-function:0 */
```

Comment:
Just wondering why this is added?

Reply:
The linter was throwing this error on line 108 due to the length of tests for `toHaveFormValues()`:

<img width="957" alt="Screen Shot 2019-08-18 at 2 56 59 PM" src="https://user-images.githubusercontent.com/10778036/63230893-8de92e80-c1c8-11e9-8df6-af0cfb472179.png">

I noticed that [this test file](https://github.com/testing-library/jest-dom/blob/master/src/__tests__/to-have-value.js) handled this problem this way, so I did the same. Let me know if another solution is preferred here!

Reply:
Ah alright, seems fair. Guess, we could split it up. I think @kentcdodds wrote an article about avoiding nesting: https://kentcdodds.com/blog/avoid-nesting-when-youre-testing